code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -38,17 +38,21 @@
 		display_text += SpecialGearRequirements()
 		to_chat(usr, display_text)
 
-/obj/item/ego_weapon/proc/CanUseEgo(mob/living/carbon/human/user)
-	if(!ishuman(user))
-		return FALSE
-	var/mob/living/carbon/human/H = user
-	for(var/atr in attribute_requirements)
-		if(attribute_requirements[atr] > get_attribute_level(H, atr))
-			to_chat(H, "<span class='notice'>You cannot use [src]!</span>")
+/obj/item/ego_weapon/proc/CanUseEgo(mob/living/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		for(var/atr in attribute_requirements)
+			if(attribute_requirements[atr] > get_attribute_level(H, atr))
+				to_chat(H, "<span class='notice'>You cannot use [src]!</span>")
+				return FALSE
+		if(!SpecialEgoCheck(H))
 			return FALSE
-	if(!SpecialEgoCheck(H))
-		return FALSE
-	return TRUE
+		return TRUE
+	if(isbot(user))
```

Comment:
If that's your concern, put it at the top instead.

Reply:
Understood